tests/README.md

```diff
@@ -1,14 +1,14 @@
 # Testing Text Processing  
 The text processing tests located in the [doc/developer/tests folder](https://github.com/mattermost/platform/tree/master/doc/developer/tests) are designed for use with the `/loadtest url` command. This command posts the raw contents of a specified .md file in the doc/developer/test folder into Mattermost.
 
-## Turning on /loadtest  
+## Turning on /test  
 Access the **System Console** from the Main Menu. Under *Service Settings* make sure that *Enable Testing* is set to `true`, then click **Save**. You may also change this setting from `config.json` by setting `”EnableTesting”: true`. Changing this setting requires a server restart to take effect.
 
 ## Running the Tests  
 In the text input box in Mattermost, type: `/loadtest url [file-name-in-testing-folder].md`. Some examples:
 
-`/loadtest url test-emoticons.md`  
-`/loadtest url test-links.md`
+`/load url test-emoticons.md`  
```

Comment:
Probably. I think I was in too much of a hurry when I did this. I still see /loadtest in a couple of places. :(